src/pages/popup/index.tsx

```diff
@@ -53,39 +54,50 @@ const CopyAction: React.FC<BoxProps> = memo(({ onClick }) => {
   );
 });
 
-const TxButton: React.FC<TxButtonProps> = memo(({ kind, onClick, ...rest }) => {
+const TxButton: React.FC<TxButtonProps> = memo(({ kind, path, ...rest }) => {
   const ref = useRef<HTMLButtonElement | null>(null);
+  const { doChangeScreen } = useAnalytics();
+  const assets = useAssets();
+
+  const isSend = kind === 'send';
+  const sendDisabled = isSend && assets.length === 0;
+
+  const handleClick = useCallback(() => {
+    doChangeScreen(path);
+  }, [path, doChangeScreen]);
```

Comment:
Maybe just delete the `path` option and have this handler simple pick the path based on `kind`? Mainly a nit